frontend/landmark-identifier/style.css

```
@@ -423,73 +628,257 @@
+  transition: color 0.2s ease;
 }
 
-@media (max-width: 720px) {
+.landmark-footer a:hover {
+  color: var(--li-accent-hover);
+  text-decoration: underline;
+}
+
+/* 10. Utilities & Animations
+   ========================================================================== */
+@keyframes fadeInUp {
```

Stylelint (17.14.0)

[error] Expected keyframe name "fadeInUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)